src/Routes.jsx

```diff
@@ -16,6 +16,7 @@ import NewsView from "./views/NewsView";
 import ArticleView from "./views/ArticleView";
 import InformationView from "./views/InformationView";
 import ErrorView from "./views/ErrorView";
+import DrupalSearchWidget from "./views/DrupalSearchWidget";
```

Comment:
I would skip the word `Drupal` in the name. Let's keep it generic

Reply:
renamed